python/cugraph-equivariant/cugraph_equivariant/_version.py

```diff
@@ -10,18 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
 
 import importlib.resources
 
-# Read VERSION file from the module that is symlinked to VERSION file
-# in the root of the repo at build time or copied to the module at
-# installation. VERSION is a separate file that allows CI build-time scripts
-# to update version info (including commit hashes) without modifying
-# source files.
 __version__ = (
-    importlib.resources.files("cugraph_equivariant")
-    .joinpath("VERSION")
-    .read_text()
-    .strip()
+    importlib.resources.files(__package__).joinpath("VERSION").read_text().strip()
 )
-__git_commit__ = ""
+try:
```

Comment:
Could you please add unit tests for all the projects that already have unit test setups, like this:

```python
import cugraph_equivariant


def test_version_constants_are_populated():
    # __git_commit__ will only be non-empty in a built distribution
    assert isinstance(import cugraph_equivariant.__git_commit__, str)

    # __version__ should always be non-empty
    assert isinstance(import cugraph_equivariant.__version__, str)
    assert len(import cugraph_equivariant.__version__) > 0
```

I did this over in `cudf` (https://github.com/rapidsai/cudf/pull/15245 and https://github.com/rapidsai/cudf/pull/15853) and from conversations offline, we'd like to roll that out with all the `rapids-build-backend` updates.